app/components/project-card-skills.js

```diff
@@ -1,21 +1,73 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  set,
+} = Ember;
+
+/**
+  `project-card-skills` composes the given project's list of skills on the
+  project's card.
+
+  ## default usage
+
+  ```Handlebars
+  {{project-card-skills skills=project.skills}}
+  ```
+
+  @class project-card-skills
+  @module Component
+  @extends Ember.Component
+ */
+export default Component.extend({
   classNames: ['project-card-skills'],
+
+  /**
+    Returns whether or not the overflow skills are hidden.
```

Comment:
@JoshSmith I think I am a little confused about what is going on here. I don't see the criteria for when there is an overflow. Also I know that we do some computed stuff around if the user has the skills, but are we just showing all the common & not common skills from this union? https://github.com/code-corps/code-corps-ember/blob/develop/app/components/skill-list-items.js#L13 

Thanks :)